synapse/handlers/room.py

```diff
@@ -1040,9 +1040,25 @@ async def send(etype: str, content: JsonDict, **kwargs: Any) -> int:
                 for invitee in invite_list:
                     power_level_content["users"][invitee] = 100
 
-            # Power levels overrides are defined per chat preset
+            # If the user supplied a preset name e.g. "private_chat",
+            # we apply that preset
             power_level_content.update(config["power_level_content_override"])
 
+            # If the server config contains default_power_level_content_override,
+            # and that contains information for this room preset, apply it.
+            default_power_level_content_override = (
+                self.config.room.default_power_level_content_override
```

Comment:
It's (unwritten) convention that for homeserver config options, we typically store them in a member variable first, and then access that member variable in the hot path. This both helps readability, as well as being perceived (though not tested) to be slightly more performant than reaching into the config dictionary.

This translates to adding a variable with a name like `self._default_power_level_content_override` to `__init__` and then using it here instead.

Reply:
Fixed in 4dda230e4